### ISSUE TITLE
403 forbidden on lhci-wizard

### ISSUE THREAD

**Issue by troydieter** [troydieter/lhci-fargate#44]
Currently hitting a `403 Forbidden` on `lhci wizard`:

```
example@DESKTOP-GNNVV9P:/mnt/c/coderepo/lhci-fargate$ lhci wizard
? Which wizard do you want to run? new-project
? What is the URL of your LHCI server? https://lighthouse.example.com
? What would you like to name the project? lhci-fargate
? Where is the project's code hosted? https://github.com/example/lhci-fargate
? What branch is considered the repo's trunk or main branch? newbranch
Error: Unexpected status code 403
  <html>
<head><title>403 Forbidden</title></head>
<body>
<center><h1>403 Forbidden</h1></center>
</body>
</html>

    at ApiClient._convertFetchResponseToReturnValue (/home/tdieter/.nvm/versions/node/v16.17.0/lib/node_modules/@lhci/cli/node_modules/@lhci/utils/src/api-client.js:79:21)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async runNewProjectWizard (/home/tdieter/.nvm/versions/node/v16.17.0/lib/node_modules/@lhci/cli/src/wizard/wizard.js:64:19)
    at async Object.runCommand (/home/tdieter/.nvm/versions/node/v16.17.0/lib/node_modules/@lhci/cli/src/wizard/wizard.js:164:7)
    at async run (/home/tdieter/.nvm/versions/node/v16.17.0/lib/node_modules/@lhci/cli/src/cli.js:125:7)
```

**Comment by jamespearson** [comment]
Came here looking for a solution to the same problem.

**Comment by troydieter** [comment]
Resolved by #50
